apps/frontend/src/features/learn/components/LearnContainer.tsx

```
@@ -42,21 +42,22 @@ export const LearnContainer = ({
-            <div css={spaceFillerStyle(isDarkMode)}></div>
+          <section css={centerSectionStyle} aria-label="유닛 및 레슨 목록">
+            <div css={spaceFillerStyle(isDarkMode)} aria-hidden="true"></div>
             {activeUnit && (
               <div css={stickyHeaderWrapperStyle} ref={headerRef}>
-                <div
+                <header
                   key={activeUnit.id}
                   css={[headerSectionStyle(), stickyHeaderStyle(theme), headerPulseStyle]}
+                  aria-label={`현재 유닛: ${activeUnit.title}`}
                 >
                   <div css={headerContentStyle}>
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)